src/ens/contract/contract.ts

```diff
@@ -0,0 +1,62 @@
+import { ResolutionError, ResolutionErrorCode } from "../..";
+import EnsProvider, { FourBytes } from "../../provider/provider";
+import BaseConnection from "../../baseConnection";
+import {defaultAbiCoder as AbiCoder} from 'ethers/utils/abi-coder';
+
+/** Contract class describes the smartContract on the etherium */
```

Comment:
Please mark this class as `@internal`. Also we do not really need the documentation for internal classes and methods..